src/test/ui/issue-53565.stderr

```diff
@@ -0,0 +1,20 @@
+error[E0432]: unresolved imports `std::time::foo`, `std::time::bar`, `std::time::buzz`
+  --> $DIR/issue-53565.rs:1:17
+   |
+LL | use std::time::{foo, bar, buzz};
+   |                 ^^^  ^^^  ^^^^ no `buzz` in `time`
```

Comment:
It might be nice for this to be

```
no `buzz` in `std::time`
```

but I'm not too worried one way or another.